src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -91,7 +93,11 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
 
     form.onExport(() -> new Thread(() -> {
       List<String> errors = export();
-      if (!errors.isEmpty()) {
+      int numberOfExportedForms = forms.getSelectedForms().size();
+      if (errors.isEmpty())
+        analytics.event("Export", "Export", "Success", numberOfExportedForms);
+      else {
+        analytics.event("Export", "Export", "Configuration errors", numberOfExportedForms);
```

Comment:
For consistency, I'd expect Failure instead of Configuration errors. Are these really configuration errors?

Also, why is this only done with export and not with pull/push?

Reply:
I wanted to differentiate when an export operation can't be done because there are configuration errors, from general export failures (which  have their our "Failure" label)

Pull and Push operations don't have (yet) the same concept of a configuration object and its validness. Once we do that, we will have the same "Configuration errors" label for them as well.

I thought that monitoring this could give us a measure of how difficult is to configure each operation. If we see lots of conf error events, maybe we need to focus on making the conf process easier...

Reply:
Oh, man! :( I was mixing in my head this PR and #353!!!!! :( :( :(

You're right. This one should be labeled "Failure". After we merge #353 we can add a new event for conf errors...

Reply:
I thought I was going crazy for a second. Please push the change and we can have the testers very this.

Reply:
I need a fresher head to review this properly... I'll do it first thing tomorrow! Sorry for the confusion!